Config-ap.gocd.yaml

```diff
@@ -144,4 +155,48 @@ pipelines:
             - docker
             tasks:
             - script: |
-                echo command to run smoke test
\ No newline at end of file
+                echo command to run smoke test
+  Capacity_testing:
+    group: AT20Group
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      GIT_COMMIT_HASH: 1.2
+      TARGET_HOST: 192.168.0.108
+    materials:
+      Config-ap:
+        pipeline: Config-ap
+        stage: PackageAndPublish
```

Comment:
It should depend on previous pipeline, if Acceptance passed, then continue with the Capacity

Reply:
I made the changes